lib/chef/provider/group/suse.rb

```diff
@@ -51,7 +52,12 @@ def set_members(members)
         end
 
         def add_member(member)
-          shell_out!("groupmod -A #{member} #{@new_resource.group_name}")
+          begin
+            Etc.getpwnam(member)
+            shell_out!("groupmod -A #{member} #{@new_resource.group_name}")
+          rescue ArgumentError => e
+            raise Chef::Exceptions::Group, "Could not add #{member} to #{@new_resource.group_name}: user does not exist"
```

Comment:
The reason I ask is I _could_ iterate through `new_resource.members` and bomb out of one of those members isn't created, but that'd only be reliable if the assertion is evaluated during the execution (i.e. the chef recipe creates the user).